docs/_docs/contributing/scaladoc.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: doc-page
+title: Scaladoc
+---
+
+Scaladoc  is the documentation tool for [Scala
+3](https://github.com/lampepfl/dotty). It uses the TastyInspector to access
+definitions, which is an officially supported way to access Scala 3's
+perspective of a codebase.
+
+We're aiming to support all the features Scaladoc did, plus new and exciting ones such as:
+
+- Markdown syntax!
+- displaying project and API documentation together on one site!
+- and more!
+
+**Yes, this page was generated using scaladoc**
+
+## Running the project
+
+Use the following commands to generate documentation for this project and for Dotty, respectively:
+
+```
+sbt scaladoc/generateSelfDocumentation
+sbt scaladoc/generateScalaDocumentation
+```
+
+To actually view the documentation, the easiest way is to run the following in the project root:
+
+```
+cd scaladoc/output
+python3 -m http.server 8080
+```
+
+And afterwards point your browser to <http://localhost:8080/self> or
+<http://localhost:8080/scala3> for this project and for Dotty documentation
+respectively.
+
+It's not strictly necessary to go through an HTTP server, but because of CORS
+the documentation won't work completely if you don't.
+
+## CLI and sbt Documentation
+
+The preferred way to use scaladoc is calling it from sbt `Compile/doc` task or to use CLI interface provided inside `dotty/bin/scaladoc` bash script.
+More information about specific scaladoc flags you can find inside [Usage docs](https://docs.scala-lang.org/scala3/guides/scaladoc/settings.html)
+
+## Developing
+
+For every PR named with prefix `scaladoc/`, we build documentation for scaladoc and Dotty. For example, for
```

Comment:
Do you mind prefixing your branch with `scaladoc/` so that we can see the result?

Reply:
I'd need to send in another pr then no?

Reply:
True… nevermind.

Reply:
Funny enough I should have thought of that because I asked about this feature when I hit on it, because I never knew it existed.